client/src/components/Comparator.js

```diff
@@ -45,20 +55,41 @@ class Comparator extends React.Component {
         )}
         {this.state.a && this.state.b && (
           <>
-            <div>
+            <div className="match">
               {Math.round(
                 this.global.result.diffs[this.state.a][this.state.b].percent *
                   100
               )}
               % match
             </div>
-            <div>
-              {this.global.result.diffs[this.state.a][this.state.b].diff
-                .split("\n")
-                .map((line, i) => (
-                  <div key={i}>{line}</div>
-                ))}
-            </div>
+            <Button
+              onClick={this.toggle}
+              style={{
+                padding: "1rem",
+                marginTop: "1.2rem",
+                fontSize: "1.6rem",
+              }}
+            >
+              More details
+            </Button>
+            <Collapse isOpen={this.state.isOpen}>
+              <div className="table">
+                {this.global.result.diffs[this.state.a][this.state.b].diff
+                  .split("\n")
+                  .map((line, i) => (
+                    <Table
+                      style={{
+                        marginTop: "1.2rem",
```

Comment:
I feel like we can set marginTop and marginBottom to 0 just so we don't need to scroll through 3 screens